pkg/master/controller.go

```diff
@@ -140,14 +137,33 @@ func (c *Controller) CreateNamespaceIfNeeded(ns string) error {
 	return err
 }
 
+// createPortsSpec creates an array of service ports which we will
+// if the NodePort value is 0, then this signifies the standard TCP port
```

Comment:
This comment needs rewording for grammar/comprehensibility. About an old version of the function?